include/sdf/Param.hh

```diff
@@ -487,26 +708,53 @@ namespace sdf
   ///////////////////////////////////////////////
   template<typename T>
   bool Param::Set(const T &_value)
+  {
+    sdf::Errors errors;
+    bool result = this->Set<T>(_value, errors);
+    for (const auto &e : errors)
+    {
+      sdferr << e.Message() + "\n";
+    }
```

Comment:
This for-loop can be shortened to `sdferr << errors << "\n";`

Reply:
True, done.

Reply:
I couldn't see the changes. Has it been pushed?